airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/errors/messages/ErrorMessage.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.base.errors.messages;
+
+import io.airbyte.integrations.base.errors.utils.ConnectionErrorType;
+import io.airbyte.integrations.base.errors.utils.ConnectorName;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public abstract class ErrorMessage {
+
+  private static final String DEFAULT_ERROR_MESSAGE = "State code: %s. \n Error Message: %s.";
+  protected final Map<String, ConnectionErrorType> CONSTANTS = new HashMap<>();
```

Comment:
the same here. Please rename CONSTANTS to ERRORCODES_TYPES

Reply:
renamed it